vscode/package.json

```diff
@@ -132,7 +132,9 @@
   "scripts": {
     "tsc:check": "node ../node_modules/typescript/bin/tsc -p ./tsconfig.json",
     "build": "npm run tsc:check && node build.mjs",
-    "run:web": "npx @vscode/test-web --extensionDevelopmentPath . ../samples"
+    "run:web": "npx --no-install @vscode/test-web --extensionDevelopmentPath . ../samples",
+    "test": "node ./test/runTest.js",
```

Comment:
I was a little confused at first by the fact this file will be side-by-side with the .ts file, but the test suite files are placed in an ./out directory. After a bit of digging around I realized they are built separately - with one building the by a package.json task, and the other being built by the build.mjs script with different options. Any way we can unify build mechanisms and locations here to make this more intuitive?

Reply:
I think I'll change this to `runTests.js` - plain JS. If I move the build to `build.mjs` it's overkill as it doesn't need to be bundled or built with ESBuild. If I make it a `tsc` build step then I have to introduce a custom step that's different than anything else in the repo.

This is just a script that's just going to run on the dev machine, so it's more along the lines of `build.mjs` or `generate_katas_content.js`. We chose plain JavaScript for those scripts, so I think I'll just follow that lead and make this JS as well.